tests/functionsTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functionsTest extends TestCase
+{
+    function testSuccessCreateRecipeCards()
+    {
+$expected = "<section class= 'recipe_card'><h2>eggs</h2><h3>Cuisine: all</h3><h3>Time (mins): 1</h3><a href= https://www.google.com/>Link to recipe</a></section>";
```

Comment:
weird indentation

Reply:
It was done by phpStorm because the line is so long